### PR TITLE
Temporarily disable shared element transitions

### LINKED ISSUE #1548: 评论区刷新闪退
### 复现步骤 / Step to reproduce

https://github.com/user-attachments/assets/2979d8dd-9c61-42e0-adfd-b4e7459ad45f

画廊链接：[https://e-hentai.org/g/3029689/021a8d0970](https://e-hentai.org/g/3029689/021a8d0970)

### 预期行为 / Expected behaviour

不闪退

### 实际行为 / Actual Behaviour

闪退

### 应用日志 / App logs

[log-2024-08-21-11-57-32-722.zip](https://github.com/user-attachments/files/16685804/log-2024-08-21-11-57-32-722.zip)

### 备注 / Addition details

评论分数阈值：`6`
评论屏蔽列表：`([彩白绿][Ee]|[Ff][Ee]|[Jj][Hh]|[Ee][Hh]|解析|加载|用不了|转圈|509|软件|[aA][pP]{2}|配额|白屏|熊猫|网络|直连|梯子|代理|兼容|导入|导出|[Gg][Ii][Tt][Hh][Uu][Bb])`

### 浏览站点 / Browsing site

E-Hentai

### EhViewer 版本号 / EhViewer version code

1.13.0-SNAPSHOT 

### EhViewer CI 版本 / EhViewer CI Version

876bb68

### Android 系统版本 / Android version

HarmonyOS 4.2.0.125

### 设备型号 / Device model

HUAWEI Nova 9 Pro

### SoC 型号 / Soc model

高通骁龙778G 4G

### 自查步骤 / Verify steps

- [X] 您已搜索过 [Issue Tracker](https://github.com/FooIbar/EhViewer/issues)，没有找到类似内容 / I have searched on [Issue Tracker](https://github.com/FooIbar/EhViewer/issues), No duplicate or related open issue has been found
- [X] 您确保这个 Issue 只提及一个问题。如果您有多个问题报告，烦请发起多个 Issue / Ensure there is only one bug report in this issue. Please make mutiply issue for mutiply bugs
- [X] 您确保已使用最新 [CI 版本](https://github.com/FooIbar/EhViewer/actions)测试，并且该问题在最新 CI 版本中并未解决 / This bug have not solved in latest [CI version](https://github.com/FooIbar/EhViewer/actions)

### PR DESCRIPTION
**Pull request by FooIbar**
https://issuetracker.google.com/336140982

Resolve #1548

### CODE REVIEW

**Comment by revonateB0T** [comment]
This is not necessary, just remove `SharedTransitionLayout` and provide a noop `SharedTransitionScope` to minimize modification.

**Comment by FooIbar** [comment]
> This is not necessary, just remove `SharedTransitionLayout` and provide a noop `SharedTransitionScope` to minimize modification.

This is to allow r8 to remove those dead code.

**Comment by revonateB0T** [comment]
> > This is not necessary, just remove `SharedTransitionLayout` and provide a noop `SharedTransitionScope` to minimize modification.
> 
> This is to allow r8 to remove those dead code.

We don't need to remove dead code as we will add it back one day.

File: app/src/main/kotlin/com/hippo/ehviewer/ui/main/EhAsyncThumb.kt [hunk 1]
```diff
@@ -57,7 +58,9 @@ fun EhAsyncCropThumb(
     AsyncImage(
         model = requestOf(key),
         contentDescription = null,
-        modifier = modifier.sharedBounds(key = "${key.gid}").clip(ShapeDefaults.Medium),
+        modifier = modifier
+            // .sharedBounds(key = "${key.gid}")
```
Comment (revonateB0T) [inline app/src/main/kotlin/com/hippo/ehviewer/ui/main/EhAsyncThumb.kt:62]: This comment out is not necessary if we return `this` in sharedBounds in NoopSharedTransitionScope

Comment (FooIbar) [reply app/src/main/kotlin/com/hippo/ehviewer/ui/main/EhAsyncThumb.kt:62]: `SharedContentState`'s constructor is internal

**Review by revonateB0T — APPROVED** [review]

**Pull request merged**